src/types.ts

```diff
@@ -10,36 +10,53 @@ export enum TransactionAddressType {
 }
 
 export type RegisterSignupProps = {
+  installationId: string
   addressCoordinates?: AddressCoordinates
   addressLine?: string
-  appId?: string
-  installationId: string
   structuredAddress?: StructuredAddress
+  accountId?: string
+  externalId?: string
+  policyId?: string
+  [x: string]: any
```

Comment:
Wild card added to allow the usage of new keys that are still not supported by the lib api.